src/js/d3_svg_cell_renderer.js

```diff
@@ -0,0 +1,207 @@
+var utils = require('./utils');
```

Comment:
Sorry, what I mean is the filename: `d3_svg_cell_renderer.js` versus `D3SVGCellRenderer.js` which is the name of the class being exported in that file. I think that if one is going to go through the effort of changing the naming conventions of the project, it should be consistent, so I didn't know if this was part of the new naming conventions.